comictaggerlib/taggerwindow.py

```diff
@@ -1017,7 +1017,10 @@ def query_online(self, autoselect: bool = False, literal: bool = False) -> None:
         # Only need this check is the source has issue level data.
         if autoselect and issue_number == "":
             QtWidgets.QMessageBox.information(
-                self, "Automatic Identify Search", "Can't auto-identify without an issue number (yet!)"
+                self,
+                "Automatic Identify Search",
+                "Can't auto-identify without an issue number. The auto-tag function has the 'If no issue number, "
+                'assume "1"\' option if desired.',
```

Comment:
quotes need to be consistent. running black or `tox r -e format` should fix it

Reply:
I think it was black that broke it after I tried to make it multi-line... Looks okay now.